azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -235,6 +240,20 @@ public KubernetesContainerizedImpl(final Props azkProps,
     }
     // Add all the job types that are readily available as part of azkaban base image.
     this.addIncludedJobTypes();
+
+    // Start the kubernetes watch if configured
+    // todo: this could benefit from a small api change in {@link ContainerizedDispatchManager}
+    //   which will also provide a suitable place for terminating the watch during server shutdown.
```

Comment:
Agree. It will be good to keep it in shutdown of executor manager.

Reply:
Good point, missed it. Will update.